05-Mining-Protocol.md

```diff
@@ -237,7 +237,7 @@ Possible error codes:
 
 ### 5.3.7 `UpdateChannel` (Client -> Server)
 Client notifies the server about changes on the specified channel.
-If a client performs device/connection aggregation (i.e. it is a proxy), it MUST send this message when downstream channels change.
+If a client performs connection aggregation (i.e. it is a proxy), it must send this message when downstream channels change.
```

Comment:
I also wouldn't change the `device/connection`, to my knowledge those are different things.